sageintacctsdk/apis/constants.py

```diff
@@ -292,5 +292,147 @@
         'MEGAENTITYKEY',
         'MEGAENTITYID',
         'MEGAENTITYNAME'
+    ],
+    'ARINVOICE': [
```

Comment:
This the error we get from Intacct if we are querying for not supported fields.
```'The following fields cannot be queried: [\n\tarinvoice: [RECORDTYPE, RAWSTATE, CUSTOMERRECORDNO, CUSTEMAILOPTIN, TRX_ENTITYDUE, DELIVERY_OPTIONS, TERMKEY, TERMVALUE, BILLTOPAYTOKEY, SHIPTORETURNTOKEY, PRBATCHKEY, MODULEKEY, SCHOPKEY, HASPOSTEDREVREC, BILLBACKTEMPLATEKEY, CREATEDBY, MODIFIEDBY, TAXSOLUTIONKEY, SHOWMULTILINETAX, SUPDOCID, EXCHRATEDATE, EXCHRATETYPE, EXCHRATE, BILLTOCONTACTNAME, SHIPTOCONTACTNAME]```

So, please remove these fields displayed in the error message.

Also, there were few missing commas (,) in the list below, please fix them too.

Reply:
If you are still interested in those not supported fields, consider writing a generic `readByQuery` in the base class.